### DISCUSSION TITLE
Header navigation Menu is working properly in mobile responsive

### DISCUSSION THREAD

**Discussion by hajanajubudeen88** [fomantic/Fomantic-UI#1015] [Collections (form, grid, menu, message, table, ...)] [answered]
# Bug Report

## Steps to reproduce
1. I created header menu its working awesome in desktop,
2. But its not working properly in mobile (May be its responsive issue),

![Capture](https://user-images.githubusercontent.com/32840808/64798599-e8479600-d5a0-11e9-8dec-4e7fe85ed8cd.JPG)

3. Without button also not working in responsive
![Capture1](https://user-images.githubusercontent.com/32840808/64798601-e8e02c80-d5a0-11e9-918d-926319e8fdff.JPG)

4. Please help to fix the issue and guide me to create the mobile menu with hamburger

## Expected result

![Capture 2](https://user-images.githubusercontent.com/32840808/64798911-715ecd00-d5a1-11e9-8e48-c5a8fc9a5bb6.JPG)

## Actual result
This image will show the exact result, <b>you can check the code in Testcase</b>
![Capture3](https://user-images.githubusercontent.com/32840808/64800382-22666700-d5a4-11e9-904b-e42caf1e6208.JPG)

## Testcase

https://jsfiddle.net/hajanajubudeen88/3dm9fr0a/7/

**Comment by lubber-de** [comment, marked as answer]
Use `stackable menu`
https://fomantic-ui.com/collections/menu.html#stackable

I adjusted your example here https://jsfiddle.net/g2m164rk/ (just add `stackable` to your menu 🙂 )

**Comment by lubber-de** [comment]
If you want "Hamburger" menu, you can make use of Sidebar module 
https://fomantic-ui.com/modules/sidebar.html
https://fomantic-ui.com/modules/sidebar.html#using-a-custom-context
